signer/src/config/default.toml

```diff
@@ -7,6 +7,23 @@
 host = "127.0.0.1"
 port = 8080
 
+# !! ==============================================================================
+# !! Emily API Configuration
+# !! ==============================================================================
+[emily]
+# The URI(s) of the Emily API server to connect to.
+#
+# You may specify multiple Emily API servers if you have them. They will be
+# tried round-robin until one succeeds.
+#
+# Format: ["http(s)://<host>:<port>", ..]
+# Default: <none>
+# Required: true
+# Environment: SIGNER_EMILY__ENDPOINTS
+endpoints = [
+    "http://localhost:3000"
+]
```

Comment:
I think Emily will have only one endpoint, but I suppose that fact can change.

Reply:
Yeah... it just pains me in my soul to code a single-point-of-failure/trust into a distributed "trustless" system :laughing: